docs/sources/developers/http_api/access_control.md

```diff
@@ -311,14 +311,80 @@ Content-Type: application/json; charset=UTF-8
 }
 ```
 
+#### Create role validation errors
+
+Permission validation only takes place when permission validation is enabled (`rbac.permission_validation_enabled = true`).
+
+> It has been enabled by default since Grafana 10.2
+
+##### Invalid action
+
```

Comment:
Please add a single introduction sentence here that describes what the reader is seeing in this section. Something like "The following example shows...".

Reply:
Done